From 847a36f064efed676ae007b1def3c0db364cdddb Mon Sep 17 00:00:00 2001
From: Sergey Poznyakoff <gray@gnu.org.ua>
Date: Tue, 13 Mar 2018 09:59:11 +0200
Subject: Testsuite bugfix

* tests/difflink.at: Define order of files within tested archive.
---
 tests/difflink.at | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/difflink.at b/tests/difflink.at
index eadfb08..4e01176 100644
--- a/tests/difflink.at
+++ b/tests/difflink.at
@@ -21,7 +21,7 @@ mkdir a
 genfile -f a/x
 ln -s x a/y
 ln a/y a/z
-tar cf a.tar a
+tar cf a.tar a/x a/y a/z
 rm a/z
 ln -s x a/z
 tar df a.tar
-- 
cgit v1.0-41-gc330

